sync_settings/libs/gist.py

```diff
@@ -63,14 +65,22 @@ def make_uri(endpoint=''):
     def create(self, data):
         if not isinstance(data, dict) or not len(data):
             raise ValueError('Gist can`t be created without data')
-        return self.__do_request('post', self.make_uri(), data=json.dumps(data)).json()
+        return self.__do_request(
+             'post',
+             self.make_uri(),
+             data=sublime.encode_value(data, True).encode('utf-8').decode('latin-1'),
```

Comment:
@mfuentesg with this line, I've done a encode -> decode due to the fact that some settings and files can cause detection of latin-1 encoding. This resolves that, it doesn't impact anything else.

Reply:
can you create a unit test for this specific case?

Reply:
reverted changes due to larger PR coming